src/store/modules/products.js

```
@@ -91,7 +91,7 @@ export default {
         const res = await fetchPicture(productId);
         return URL.createObjectURL(new Blob([res], { type: "image/png" }));
       } catch (error) {
-        throw new Error('Could not upload the picture!')
+        return null;
```

Can we return the error here? It might be useful to the caller function.